.github/workflows/ci.yml

```diff
@@ -192,8 +196,12 @@ jobs:
       - name: Start mysql
         run: sudo systemctl start mysql
 
-      - name: Run unit tests
-        run: bundle exec rake test:multiverse[group=${{ matrix.multiverse }},verbose]
+      - name: Run Multiverse Tests
+        uses: nick-invision/retry@v1.0.0
+        with:
+          timeout_minutes: 20
```

Comment:
I noticed this caused one of the multiverse test runs to time out. Should probably be bumped up here.